tests/subsys/pm/device_dependencies/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

should be update to 3.10 now.
```suggestion
cmake_minimum_required(VERSION 3.20.0)
```